docs/contributors/code/auto-cherry-picking.md

```diff
@@ -0,0 +1,73 @@
+# Cherry-picking automation
+
+The cherry-picking automation makes cherry-picking PRs into WordPress and Gutenberg release branches a breeze.
+
+It works as follows: you switch to the relevant release branch and run  `npm run cherry-pick`.
+
+It gives you the following prompt:
+
+```
+You are on branch "wp/6.2".
+This script will:
+• Cherry-pick the merged PRs labeled as "Backport to WP Beta/RC" to this branch
+• Ask whether you want to push this branch
+• Comment on each PR
+• Remove the label from each PR
+
+The last two actions will be performed USING YOUR GITHUB ACCOUNT that you've
+linked to your GitHub CLI (gh command)
+
+Do you want to proceed? (Y/n)
+```
+
+Here's what happens once you agree:
+
+```
+Trying to cherry-pick one by one..
+
+$ git pull origin wp/6.0 --rebase...
```

Comment:
Shouldn't this be

```suggestion
$ git pull origin wp/6.2 --rebase...
```

to be consistent with [R10](https://github.com/WordPress/gutenberg/pull/42932/files#diff-4ece3a7cc17b7f8353fe71247bdc53d499a0024a1f5efe63d71c92caf3c55dc6R10)?